rakefile.rb

```diff
@@ -95,6 +101,9 @@
 desc "Execute default tasks"
 task :default => [ :vars, :gitlink, :unit, :integ, :spec, :pack ]
 
+desc "Execute default tasks plus .NET Core tasks"
+task :dotnet => [ :default, :unit_dotnet, :pack_dotnet]

```

Comment:
And all the `_dotnet` packages will build/test/pack both the .NET Platform Standard and the .NET 4.0 versions? That would seem to follow along from your suggestion that we be able to complete the whole build by calling the single `dotnet` target.